### DISCUSSION TITLE
Deploy custom trusted site list to users using Edge Browser

### DISCUSSION THREAD

**Discussion by TayMC241** [uBlockOrigin/uBlock-issues#3214] [Q&A]
Hi!
Is there a way to deploy a central whitelist to all of our uBlock colleagues using Microsoft Edge?  There are about 8 domains I'd like to automatically be whitelisted on the uBlock plugin, and would like to avoid users having to manually add them to their individual whitelists.    We are using uBlock version 1.57.2 and Windows 10/11 OS.
Thanks!

**Comment by stephenhawk8054** [comment]
You can find some guides here: https://github.com/gorhill/uBlock/wiki/Deploying-uBlock-Origin

**Reply by TayMC241** [reply]
Thank you, I'll check this out.

**Reply by stephenhawk8054** [reply]
Here's the configuration branch that's related to trusted sites: https://github.com/gorhill/uBlock/wiki/Deploying-uBlock-Origin:-configuration#toadd

**Reply by TayMC241** [reply]
Thanks again.  I followed https://www.reddit.com/r/uBlockOrigin/comments/o7q2ou/comment/h3cplhd/ from within the link you provided and confirmed it works, I'm able to predefine the whitelist in my Ublock settings and deploy via powershell.
